src/Network/Network/ChangeLog.md

```diff
@@ -21,9 +21,16 @@
 
 ## Version 1.1.0
 * Update incorrect online help URLs
+* Added Cmdlets for Identity on Application Gateway.
+    - New cmdlets added:
+        - Set-AzApplicationGatewayIdentity
+        - Get-AzApplicationGatewayIdentity
+        - New-AzApplicationGatewayIdentity
+        - Remove-AzApplicationGatewayIdentity
+    - New-AzApplicationGateway cmdlet updated with optional parameter -Identity
+>>>>>>> change log
 
 ## Version 1.0.0
-* General availability of `Az.Network` module
```

Comment:
@akshaysngupta please revert the change made to this line

Reply:
corrected.

Reply:
@akshaysngupta same comment -- no new commit, so this is still an issue